labs/lab-1-html-and-css/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <head>
+      <title>chirpus inc.</title>
+      <link href="style-pt-2.css" rel="stylesheet" type="text/css">
+      <meta name="viewport" content="width=device width, initial-scale=1">
+      <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,800" rel="stylesheet">
+    </head>
+  </head>
+
+  <body>
+
+    <div class="logo">chirpus inc.</div>
+
+    <nav>
+      <a class="nav" href="index.html">Home</a>
```

Comment:
`nav-link` would be a better class name here; right now, when I saw `.nav {` in your css, I thought it was a mistake; it's like seeing `.div {`